acs-apiclient/Droid/Activity/ExternalFlowWebActivity.cs

```diff
@@ -0,0 +1,130 @@
+﻿
+using System;
+using Android.App;
+using Android.OS;
+using Android.Views;
+using Android.Webkit;
+using acs_apiclient.Droid.CustomViews;
+using Android.Widget;
+using System.Threading;
+using AcsApi;
+using Android.Graphics;
+
+namespace acs_apiclient.Droid
+{
+    [Activity(Label = "ExternalFlowWebActivity", 
+    Theme = "@android:style/Theme.NoTitleBar", 
+    WindowSoftInputMode = SoftInput.AdjustResize)]
+    public class ExternalFlowWebActivity : Activity
+    {
+        public static string UrlParamKey = "url";
+        private static Android.Graphics.Color Obsidian = Android.Graphics.Color.ParseColor("#222b3c");
+        private static Android.Graphics.Color Ash = Android.Graphics.Color.ParseColor("#B7C0CC");
+        private static Android.Graphics.Color SelectedButtonTintColor = Ash;
+        private static Android.Graphics.Color UnselectedButtonTintColor = Obsidian;
```

Comment:
💡 Can these be pulled into shared code?

Reply:
Pulled into new Constants file under `Colors`

Reply:
✅